lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/EvaluatorRequestorImpl.java

```diff
@@ -72,7 +72,7 @@ public synchronized void submit(final EvaluatorRequest req) {
     }
 
     try (LoggingScope ls = loggingScopeFactory.evaluatorSubmit(req.getNumber())) {
-      final DriverRuntimeProtocol.ResourceRequestProto.Builder request = DriverRuntimeProtocol.ResourceRequestProto
+      final ResourceRequestEventImpl.Builder request = ResourceRequestEventImpl
```

Comment:
`EvaluatorRequest` is missing some fields, such as `priority` and `relaxLocality` that are added inside the runtimes. It's possible that we actually want these exposed to the user and not the runtime. But I want to defer that exploration to REEF-69.

Also, using `EvaluatorRequest` breaks the symmetry of  `Resource...Event` runtime events.

@markusweimer how does that sound?